kolibri/core/content/api.py

```diff
@@ -210,6 +211,11 @@ class OptionalPageNumberPagination(pagination.PageNumberPagination):
     page_size_query_param = "page_size"
 
 
+class SQSum(Subquery):
+    template = "(SELECT SUM(\"%(field)s\") FROM (%(subquery)s))"
```

Comment:
Saw that you can also pass in a kwarg of template but I do think this way is cleaner!